nautilus_trader/common/actor.pyx

```diff
@@ -2142,6 +2156,8 @@ cdef class Actor(Component):
         datetime end = None,
         ClientId client_id = None,
         callback: Callable[[UUID4], None] | None = None,
+        bint update_catalog = False,
+        str quote_type = "",
```

Comment:
Is this to specify the bbo schemas?

I think this is fine, probably one of the only ways to do it without adding specific data types, which might be overkill.